packages/webview_flutter/webview_flutter_wkwebview/example/lib/main.dart

```diff
@@ -352,3 +363,584 @@ class NavigationControls extends StatelessWidget {
     );
   }
 }
+
+/// A web view widget for showing html content.
```

Comment:
Let's put these classes that wouldn't be part of normal webview package use into a separate file, and put a comment at the top explaining why they are needed, since I don't think it'll be obvious to someone new to the strucutre why this is done.

Reply:
I have put all classes in one file on purpose since the contents of the `example/main.dart` file is shown on the "Example" tab on pub.dev. In the past I have received complaints from users explaining that the code shown on this page doesn't at all demonstrates how to use the plugin since that code is hidden in other files which are not displayed. Since the actual usage of the `webview_flutter_android` and `webview_flutter_wkwebview` is demonstrated in the `WebView` widget (which has been separated from the `WebViewExample` widget to facilitate integration testing) moving it out into a separate file means the pub.dev "Example" page would not show how to use/ implement the platform specific package. 

Before I start moving the classes out, I would like to double check if you are fine with these consequences? Most likely the platform specific packages will not be used directly by developers (and the packages should be marked as unlisted on pub.dev)  but it doesn't hurt to double check and be sure ;).

Reply:
> In the past I have received complaints from users explaining that the code shown on this page doesn't demonstrate how to use the plugin because that particular code is hidden in files that are not displayed on the "Example" tab on pub.dev.
> [...]
> Most likely the platform specific packages will not be used directly by developers (and the packages should be marked as unlisted on pub.dev) but it doesn't hurt to double check and be sure ;).

Yes, I don't expect anyone to actually use these packages directly; the example exists to run integration tests with. If someone *really* want to use the implementation package directly by itself, and are already going to the trouble of manually constructing pub.dev URLs to go to the unlisted page for it, they can take the extra step to click to the repo.

Reply:
Great thank you for confirming, I will continue with the necessary changes.

Reply:
Done.